app/templates/common/base_main.html

```diff
@@ -115,6 +115,143 @@ <h6>The user agrees not to:</h6>
+            <img src="/static/img/Data_search_page.png"alt="Portal_access_from_conp.ca" width="700px">
+            <p>Edit the code in the window to the left, and click "Run" to view the result.</p>
+            <p>Clicking on a dataset row in the Data search page will bring the user to a more 
+            detailed dataset page that includes more description information pertinent to the 
+            dataset as well as download instructions to follow using DataLad.</p>
+            <img src="/static/img/Dataset_detailed_page.png"alt="Portal_access_from_conp.ca" width="700px">
+            <p>Note that some datasets may require a third-party account. Instructions on how to 
+            obtain an account for those datasets will be displayed on the dataset page if that is a requirement.</p>
+            <p>Below are two dataset examples, one that requires a third-party account (PREVENT-AD 
+            open data) and one that does not require any account to download it (BigBrain dataset).</p>
+            <img src="/static/img/Dataset_accounts.png"alt="Portal_access_from_conp.ca" width="700px">
+            <h5>How to download the data?</h5>
+            <p>Dataset download is available through DataLad. Each dataset page contains a detailed 
+            ‘Dataset Download Instructions’ section that lay out the different steps to perform in order 
+            to download the dataset.</p>
+            <img src="/static/img/Dataset_download_instructions.png"alt="Portal_access_from_conp.ca" width="700px">
+          </ul>
+      </div>
+      <div class="modal-footer">
+        <button type="button" class="btn btn-primary" id="btnAcceptTut2">Help</button>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="modal fade" id="tutModal3" tabindex="-1" role="dialog" data-backdrop="static" data-keyboard="false"
+  aria-labelledby="modalCenterTitle" aria-hidden="true">
+  <div class="modal-dialog modal-dialog-centered modal-lg" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h4 class="modal-title w-100 text-center" id="modalLongTitle">Help</h4>
+      </div>
+      <div class="modal-body">
+        <ul>
+            <h4>How can I find and run tools/pipelines?</h4>
+            <h5>How to find a tool/pipeline?</h5>
+            <p>The Tools & Pipelines page lists all softwares registered in the portal. Each tool 
+            or pipeline present in that list page contains basic information about the tool/pipeline 
+            and its pipeline ID.</p>
+            <img src="/static/img/Pipeline_search_page.png"alt="Portal_access_from_conp.ca" width="700px">
+            <p>Clicking on a specific choice will bring the user to a more detailed page with links 
+            to instructions on how to run the tool locally, to HCPs or to external platforms.</p>
+            <img src="/static/img/Pipeline_detailed_page.png"alt="Portal_access_from_conp.ca" width="700px">
```

Comment:
Same general comment as for the dataset section. This should be divided in two: one tools & pipelines search to be displayed on this page and one specific to the pipeline page to be displayed on the pipeline page.

```suggestion
            <h4>Tools & Pipelines Search Page</h4>
            <p>The Tools & Pipelines Search page lists all softwares registered in the portal. Each tool 
            or pipeline present in that list page contains basic information about the tool/pipeline 
            and its pipeline ID.</p>
            <img src="/static/img/Pipeline_search_page.png"alt="Portal_access_from_conp.ca" width="700px">
            <p>Clicking on a specific choice will bring the user to a more detailed page with links 
            to instructions on how to run the tool locally, to HCPs or to external platforms.</p>
            <img src="/static/img/Pipeline_detailed_page.png"alt="Portal_access_from_conp.ca" width="700px">
```